content/docs/reference/install.md

```diff
@@ -127,7 +136,7 @@ The installer script can be rerun to subsequently install new updates.
 
 <div id="linux_installation">
 {{% md %}}
-## Linux
+### Installation script
```

Comment:
Should "script" be capitalized or not? It's upper case in the macOS section above but lower case in the sections below. Same question about "installation".

Reply:
I like referring to [Google's style guide](https://developers.google.com/style/capitalization) and have gotten used to sentence case, but I do notice we have mostly used title case previously. We can have a further chat around this, and I can revert to using title case for now.

Reply:
Sounds good. I don't have a preference either way, as long as we're consistent on a page (and across the site).

Reply:
I reverted for now and will apply the site-wide change later. As per [this article](https://www.herosmyth.com/article/headline-capitalization-ux-title-case-vs-sentence-case), sentence case is apparently more "casual" and is geared towards younger people. ;) So, I think I addressed all your feedback. I'd keep this branch for more "Get Started" follow-ups but  perhaps we can have this PR merged if all the changes look good now?